redaxo/src/addons/structure/plugins/content/lib/article_slice.php

```diff
@@ -220,8 +223,9 @@ public function getNextSlice($ignoreOfflines = false)
     public function getPreviousSlice($ignoreOfflines = false)
     {
         return self::getSliceWhere(
-            'priority = ? AND article_id=? AND clang_id = ? AND ctype_id = ? AND revision=?'.($ignoreOfflines ? ' AND status = 1' : ''),
-            [$this->_priority - 1, $this->_article_id, $this->_clang, $this->_ctype, $this->_revision]
+            'priority '.($ignoreOfflines ? '<=' : '=').' ? AND article_id=? AND clang_id = ? AND ctype_id = ? AND revision=?'.($ignoreOfflines ? ' AND status = 1' : ''),
```

Comment:
Konstanten für status einführen?

Reply:
Passt für mich nicht hier in den PR 😛 (der Part ändert sich ja nicht)